### DISCUSSION TITLE
Clearing "For you" and "Following"

### DISCUSSION THREAD

**Discussion by konhi** [community/community#14620] [Other Feature Feedback, Questions, & Ideas]
I'm very happy about new feed features. It puts **Explore** and **For You** into one place + makes it based on my interests!

I suggest adding a feature that would allow users to clear theirs **"For you"** and **"Following"** sections. It would allow users to **filter out** cards they've already seen before. In result, result in these sections would be more:
- appealing
- productive
- fresh

Examples:
- [TweetDeck](https://tweetdeck.twitter.com/)
- e-mails

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
